src/common/models/Projection.ts

```diff
@@ -210,6 +214,9 @@ export class Projection {
 
     this.contractTracers = this.calcContactTracers(this.actualTimeseries);
 
+    this.caseDensityCases = this.calcCaseDensityCases(this.actualTimeseries);
+    this.caseDensityDeaths = this.calcCaseDensityDeaths(this.actualTimeseries);
```

Comment:
We'll probably also need to add something like:

```
    this.currentCaseDenistyByCases = this.lastValue(this.caseDensityByCases);
    this.currentCaseDenistyByDeaths = this.lastValue(this.caseDensityByDeaths);
```

So that we can get the current values for use in the actual metric value and perhaps in the status text above the chart, etc.